internal/binary.bzl

```diff
@@ -57,12 +57,17 @@ def postcss_binary(
     for key, value in plugins.items():
         plugins_keyed_by_infos["%s.info" % key] = value
 
+    kwargs_always_private = dict()
+    for (key, value) in kwargs.items():
+        kwargs_always_private[key] = value
+    kwargs_always_private.setdefault("visibility", ["//visibility:private"])
```

Comment:
This variable's name seems to suggest that visibility should always be private, but `setdefault` lets an existing value take precedence. Either way, consider using [`dicts.add()`](https://github.com/bazelbuild/bazel-skylib/blob/master/docs/dicts_doc.md#dictsadd) to make this clearer and more concise.

Reply:
Oops, thanks for pointing out the mistake!

The documentation seems to be confusing on `dicts.add(dictionaries, kwargs)` - why is there kwargs if dictionaries exists?

If I understand correctly it would be called like `dicts.add([kwargs, {visibility: ["//visibility:private"]}])`.

I've made a commit with an alternative approach `**{k: kwargs.get(k) for k in kwargs if k not in ["visibility"]}` that doesn't add a dep on skylib.

It's not pretty either but for adding skylib dep + the former, vs the latter approach, both don't seem that nice - WDYT?

Reply:
Hang on I called this too early - I do like the skylib approach

Reply:
Done.